api/test/e2e/global_rule_test.go

```diff
@@ -56,7 +56,6 @@ func TestGlobalRule(t *testing.T) {
 			Path:   "/apisix/admin/global_rules/1",
 			Method: http.MethodPut,
 			Body: `{
-                                "id": "1",
```

Comment:
Why this line should be deleted?

Reply:
We support no ID in body. Deleted to test this scenario